docs/docs/1.2.1/deployments/multi-zone.md

```diff
@@ -132,6 +132,11 @@ $ KUMA_MODE=zone \
 
 Where `<zone-name>` is the name of the zone matching one of the Zone resources to be created at the Global CP. `<global-zone-sync-address>` is the public address as obtained during the Global CP deployment step.
 
+::: warning
+In the example above Zone CP is running with in-memory type of store. It's not possible to scale Zone CP while using in-memory store.
+Please consider using Postgres.
+:::
+
```

Comment:
```suggestion
::: warning
In the example above Zone CP is running with in-memory type of store. It's not possible to scale Zone CP while using in-memory store.
Please consider using Postgres.
:::
Note that this example runs the zone control plane with an in-memory datastore. This approach is not recommended for product because it does not scale. Consider running with Postgres instead.
```